contracts/Interfaces/IERC20Indexable.sol

```diff
@@ -26,19 +26,19 @@ interface IERC20Indexable is IERC20, IPositionManagerDependent {
     function INDEX_PRECISION() external view returns (uint256);
 
     /// @return Current index value.
-    function currentIndex() external view returns (uint256);
+    function currentIndex() external view virtual returns (uint256);
```

Comment:
probably no need for virtual in interface

Reply:
Yeah, wanted to write that :D